lib/maths/CModelStateSerialiser.cc

```diff
@@ -9,6 +9,7 @@
 #include <maths/CTimeSeriesModel.h>
 
 #include <boost/bind.hpp>
+#include <boost/make_unique.hpp>
```

Comment:
superflous? this class does not use make_unique

Reply:
Although it wasn't, this is now being used (see [this](https://github.com/elastic/ml-cpp/pull/108/commits/f3b30df1c66a54e1f2ca570f975e01e8abb4b760) commit).